CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.6.3)
+project(kvssignaling C)
+
+option(SIGNALING_DO_NOT_USE_CUSTOM_CONFIG "Do not have custom signaling config file" OFF)
```

Comment:
Could you invert the variable so we don't need to deal with double not's?

Reply:
+1

Reply:
This option is removed at latest version. Thank you.